typing/ctype.mli

```diff
@@ -176,18 +176,9 @@ val generic_instance: type_expr -> type_expr
 val instance_list: type_expr list -> type_expr list
         (* Take an instance of a list of type schemes *)
 val new_local_type:
-<<<<<<< HEAD
         ?loc:Location.t -> ?manifest_and_scope:(type_expr * int) ->
         Jkind.t -> jkind_annot:Jkind.annotation option -> type_declaration
 val existential_name: constructor_description -> type_expr -> string
-||||||| 121bedcfd2
-        ?loc:Location.t ->
-        ?manifest_and_scope:(type_expr * int) -> unit -> type_declaration
-val existential_name: constructor_description -> type_expr -> string
-=======
-        ?loc:Location.t ->
-        ?manifest_and_scope:(type_expr * int) ->
-        type_origin -> type_declaration
```

Comment:
I'm skeptical about the dropping of `type_origin` and think we should add it to our `new_local_type` function signature for now.